VM/src/lapi.cpp

```diff
@@ -1376,6 +1376,16 @@ void lua_clonefunction(lua_State* L, int idx)
     api_incr_top(L);
 }
 
+void lua_cleartable(lua_State* L, int idx)
+{
+    StkId t = index2addr(L, idx);
+    api_check(L, ttistable(t));
+    Table* tt = hvalue(t);
+    if (tt->readonly)
+        luaG_runerror(L, "Attempt to modify a readonly table");
```

Comment:
I missed this in rawsetfield, but this should use luaG_readonlyerror

Reply:
I can swap this out during the merge I guess, since rawsetfield needs a fix as well

Reply:
Ah, I thought I copied the read-only check from another API function. Anyway, thanks for fixing it!